README.md

```diff
@@ -1,2 +1,142 @@
-# metrics-webapp
-A website for displaying information about the currency of a country and its conversion rate to other currencies .
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>METRICS-WEBAPP README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [METRICS-WEBAPP] <a name="about-project"></a>
+
+**[Metrics Webapp]** is a website that display a list of cryptocurrency coins and their current market info.
+
+## 🛠 Built With <a name="built-with">React.js</a>
+
+### Tech Stack <a name="tech-stack">CSS</a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">REACT JS </a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Display a list of Cryptocurrency coins]**
+- **[A search input to make searching for coins easier]**
+- **[A green or red icon to show if coin is trading higher or not]**
+- **[State is managed with Redux]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo](metrics-website.netlify.app)
+
```

Comment:
Hey @Otegaa 👋  you have done a great job on this project but there is a problem. The link to your live demo in the README file is broken. Kindly fix this.

> Screenshot
<img width="1162" alt="Screenshot 2023-05-26 at 10 45 03" src="https://github.com/Otegaa/metrics-webapp/assets/71140133/26bfa8ac-c865-47cb-965b-f708e7444f32">